report-pr-errors

```diff
@@ -116,7 +116,7 @@ class Logs(object):
             globbed.extend(glob(sp))
 
         suffix = ("latest" + "-" + self.develPrefix).strip("-")
-        logs = [x for x in globbed if dirname(x).endswith(suffix)]
+        logs = { x for x in globbed if dirname(x).endswith(suffix) }
```

Comment:
why the extra spaces?

Reply:
A collateral damage of an unwanted removal of a virtualenv that contained my python formatter ;-)  Now fixed.